examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1002,10 +1002,10 @@ impl FrontendNode {
         expect_eq!(3, buffer.len());
         expect_eq!(0, handles.len());
 
-        // Now expect the channel status to be orphaned.
+        // Now the channel status is orphaned, causing wait_on_channels to fail
         expect_eq!(
-            vec![ChannelReadStatus::Orphaned, ChannelReadStatus::ReadReady],
-            status_convert(oak::wait_on_channels(&[in1, in2]))?
+            Err(OakStatus::ErrBadHandle),
```

Comment:
(Maybe you've not gotten to it yet, but I think the SDK function will need a change of signature too, to allow the `[Orphaned, ReadReady]` vec to still be accessible here.)